h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeNode.java

```diff
@@ -549,6 +577,19 @@ public final int getSplitIndex() {
   public final int next(double[] value) {
     final double d = value[colId];
 
+    if (
```

Comment:
is this just reformatted code from the few lines bellow?

Reply:
yes, this is just me tryinng to undrestand things

Reply:
i find this notation more readable and would like to keep it

Reply:
sure